elements/rh-tile/demo/link-with-icon.html

```diff
@@ -0,0 +1,23 @@
+<rh-tile compact desaturated>
+  <h2 slot="headline"><a href="#top">Compact link tile</a></h2>
+  <rh-icon slot="icon" set="standard" icon="mug"></rh-icon>
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit set est egestat, a sollicitudn mauris tincidunt.</p>
+</rh-tile>
+
+<script type="module">
+  import '@rhds/elements/rh-tile/rh-tile.js';
+</script>
+
+<link rel="stylesheet" href="../rh-tile-lightdom.css">
+
+<style>
+  rh-tile {
+    max-width: 360px;
+    margin-inline-end: var(--rh-space-md, 8px);
```

Comment:
Since this is a specced/default layout...IMO, the margin/gap should be handled internally or in `rh-tile-lightdom.css`. It's not a pattern.